iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -54,6 +54,7 @@ UnixDomainSocket::UnixDomainSocket(UnixDomainSocket&& other) noexcept
     *this = std::move(other);
 }
 
+// NOLINTNEXTLINE(readability-function-size) todo: make a struct out of aruments
```

Comment:
I don't know if we have this rule but lately we usually do not just use a `todo` but also add a corresponding issue number

Reply:
Added #832 as ticket number and for later refactoring